### PR TITLE
Update insight CLI docs to improbve ingress support

### PR DESCRIPTION
**Pull request by SebastianOsuna**
Which other branches should this be merged with (if any)?

### CODE REVIEW

File: scst-store/using-encryption-and-connection.md [hunk 1]
```diff
@@ -1,111 +1,79 @@
...
-    targetPort: PORT
-...
-status:
-  loadBalancer:
-    ingress:
-    - ip: IP-ADDRESS
-```
-
-Where:
+## <a id="ingress"></a>Using `Ingress`
```
Comment (ohlookadollar) [inline scst-store/using-encryption-and-connection.md:11]: nitpick: should this be wrapped in the <a> tag like above?

File: scst-store/using-encryption-and-connection.md [hunk 2]
```diff
@@ -1,111 +1,79 @@
...
 
-The connection to the Store requires TLS encryption. Use the following instructions to set up the TLS connection, depending on which type of service you are using:
+The connection to the Store requires TLS encryption, the configuration depends on the kind of installation. Use the following instructions to set up the TLS connection, depending on which type setup you are using:
 
-1. Using `LoadBalancer`
-1. Using `NodePort` — commonly used with local clusters such as kind, or minikube
+1. <a href="#ingress">Using `Ingress`</a>
+1. <a href="#no-ingress">Without `Ingress`</a>
+    - <a href="#use-lb">Using `LoadBalancer`</a>
+    - Using `NodePort` — commonly used with local clusters such as kind or minikube
```
Comment (ohlookadollar) [inline scst-store/using-encryption-and-connection.md]: Should there be a link for nodeport?

File: scst-store/using-encryption-and-connection.md [hunk 3]
```diff
@@ -119,8 +87,19 @@ Run this command in a separate terminal window.
...
+# delete any previously added entry
+sudo sed -i '' "/$METADATA_STORE_DOMAIN/d" /etc/hosts
+
+echo "127.0.0.1 $METADATA_STORE_DOMAIN" | sudo tee -a /etc/hosts > /dev/null
+```
+
+Finally, set the target 
+
+```bash
+tanzu insight config set-target https://$METADATA_STORE_DOMAIN --ca-cert insight-ca.crt
```
Comment (ohlookadollar) [inline scst-store/using-encryption-and-connection.md]: we should specify a port with the url. In the port forwarding section you are forwarding 8443

File: scst-store/using-encryption-and-connection.md [hunk 4]
```diff
@@ -1,111 +1,79 @@
 # Configure target endpoint and certificate
 
-The connection to the Store requires TLS encryption. Use the following instructions to set up the TLS connection, depending on which type of service you are using:
+The connection to the Store requires TLS encryption, the configuration depends on the kind of installation. Use the following instructions to set up the TLS connection, depending on which type setup you are using:
```
Comment (Pivotal-Jeff-Jun) [inline scst-store/using-encryption-and-connection.md]: "... depending on which `type of setup` you are using..."

File: scst-store/using-encryption-and-connection.md [hunk 5]
```diff
@@ -1,111 +1,79 @@
...
+Additionally, the endpoint host will be set to `metadata-store.<ingress-domain>` (ex: `metadata-store.example.domain.com`), depending on the value of `ingress_domain` set during the setup.
 
-```console
-kubectl get secret ingress-cert -n metadata-store -o json | jq -r '.data."ca.crt"' | base64 -d > PATH
-```
+If no accessible DNS record exists for such domain, you can edit your `/etc/hosts` file to add a local record.
 
-### <a id='dns-resolution'></a>DNS resolution
+```bash
+ENVOY_IP=$(kubectl get svc envoy -n tanzu-system-ingress -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
```
Comment (xtreme-debbie-chen) [inline scst-store/using-encryption-and-connection.md:26]: Just out of curiosity, is `tanzu-system-ingress` the default namespace for contour when it's installed through TAP?

File: scst-store/using-encryption-and-connection.md [hunk 6]
```diff
@@ -1,111 +1,79 @@
 # Configure target endpoint and certificate
 
-The connection to the Store requires TLS encryption. Use the following instructions to set up the TLS connection, depending on which type of service you are using:
+The connection to the Store requires TLS encryption, the configuration depends on the kind of installation. Use the following instructions to set up the TLS connection, depending on which type of setup you are using:
```
Comment (xtreme-jason-smith) [inline scst-store/using-encryption-and-connection.md:3]: Would it be worthwhile to mention somewhere in the doc the suggested configuration for the installation kind?

For example: kind cluster - nodeport, GKE - LB, etc...?

Comment (xtreme-jason-smith) [reply scst-store/using-encryption-and-connection.md:3]: Never mind, I see that Nodeport has examples.  But when should a user pick Ingress vs LB? Do we have recommendations?

File: scst-store/using-encryption-and-connection.md (hunk 5 above)
Comment (SebastianOsuna) [reply scst-store/using-encryption-and-connection.md:26]: yes, it is TAP's default contour installation

File: scst-store/using-encryption-and-connection.md (hunk 6 above)
Comment (SebastianOsuna) [reply scst-store/using-encryption-and-connection.md:3]: That would be up to the user, but Ingress is always recommended whenever possible, and that's part of another doc story incoming to update [multicluster setup docs](https://docs-staging.vmware.com/en/draft/Tanzu-Application-Platform/1.1/tap/GUID-multicluster-getting-started.html)

**Review by Pivotal-Jeff-Jun — APPROVED** [review]

**Review by xtreme-debbie-chen — APPROVED** [review]

**Pull request merged**